src/fiatconnect/TransferStatusScreen.tsx

```diff
@@ -200,11 +201,13 @@ export default function FiatConnectTransferStatusScreen({ route, navigation }: P
     navigateHome()
   }
 
-  // add loading description if sending is taking a while
-  const [loadingDescription, setLoadingDescription] = useState('')
+  // make loading description visible if sending is taking a while
+  const [loadingDescriptionColor, setLoadingDescriptionColor] = useState(
```

Comment:
I wanted to avoid hardcoding the "invisible" color to white, since we may support dark mode in the future